### ISSUE TITLE
When training with cifar10 dataset, the d_loss goes to 0 very fast

### ISSUE THREAD

**Issue by zj45** [taki0112/Self-Attention-GAN-Tensorflow#6]
Maybe the discriminator is too powerful? Tried plenty of learning rate settings but all of them don't work.
Do you have any tips?

**Comment by taki0112** [comment]
I fix it

**Comment by aashish-kumar** [comment]
@taki0112 Similar problem with mnist
